src/modules/assertions/require_max_file_number.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  requireEvent,
+  requirePr
+} from "#/assertions";
+import { github } from "src/infra/github";
+
+export const  requireMaxFileNumber = async () => {
+  const max_files_allowed = 100;
+  requireEvent();
+  const pr = await requirePr();
```

Comment:
Rather than fetching the PR number twice, we should fetch it once and pass it as a parameter to this function.
```suggestion
export const  requireMaxFileNumber = async (pr: PR) => {
  const max_files_allowed = 100;
  requireEvent();
```
Note, the call site will also need to change, and `import { PR } from "src/domain";` will need to be added.

Reply:
Roger that